scripts/install_plugin_modules.js

```diff
@@ -78,13 +80,18 @@ function assertFolder (name, version) {
 }
 
 function assertPackage (name, version, dependency) {
+  let additionalDependencies
+  if (dependencies[name]) {
+    additionalDependencies = dependencies[name][version]
+  }
   fs.writeFileSync(filename(name, version, 'package.json'), JSON.stringify({
     name: [name, sha1(name).substr(0, 8), sha1(version)].filter(val => val).join('-'),
     version: '1.0.0',
     license: 'BSD-3-Clause',
     private: true,
-    optionalDependencies: {
-      [name]: dependency
+    dependencies: {
+      [name]: dependency,
+      ...additionalDependencies
```

Comment:
wouldn't this break node 4 plugin tests?

Reply:
It would but I thought we run tests in >=node8. I think I've seen a lot of ES6 stuff in tests. I can remove it.

Reply:
Actually this wouldn't affect the tests themselves. This is only invoked to install/setup the dependencies, not to run the tests. We are already using es6 features like let/const in these scripts so I suppose the script was never intended to be used with node 4?  In any case, I've removed the spread operator.